docs/release_notes/0.92.0.md

```diff
@@ -0,0 +1,9 @@
+# Release 0.92.0
+
+## Features
+
+- Add support for EKS 1.22 (#4915)
+
+## Bug Fixes
+
+- remove redundant version checks, including EKS platform version (#5055)
```

Comment:
We should use a title similar to the [linked issue](https://github.com/weaveworks/eksctl/issues/4997) because this has no info on what the bug fix was. I know we try to avoid editing the title but we missed this in the PR.

Reply:
@aclevername I have zero context. What was the fix here? Do you have a better title?

Reply:
add a suggestion